### DISCUSSION TITLE
Configuration file support

### DISCUSSION THREAD

**Discussion by deekshithkv** [cesanta/mongoose#1972] [Ask questions, get answers] [answered]
Hi, I am looking for a webserver with configuration file support so that i am edit the config file and restarting webserver will do the changes needed, instead of rebuilding the app. Does mongoose have configuration file for handling address, port etc. Or is there a way to add one. I went through the tutorials i didn't find one. Please help me to get an answer.

**Comment by scaprile** [comment, marked as answer]
Mongoose is a Network Library, you write the code for your needs. The closest to what you ask is the closest example to what you use: https://github.com/cesanta/mongoose/tree/master/examples/http-server
That one parses input arguments. Again, you code what you need,
